### PR TITLE
Added removal of navigational elements

### PR DESCRIPTION
**Pull request by ShakedEdr1**

### CODE REVIEW

File: src/view/pages/statement/components/SimilarStatementsSuggestion/StepOneStatementInput.tsx [hunk 1]
```diff
@@ -131,6 +130,22 @@ export default function StepOneStatementInput({
...
+							buttonBackground="var(--button-blue)"
+							buttonWidth="10rem"
+							circlePos="relative"
+							circleRight="1rem"
+							circleBorder="2px solid var(--white)"
+							circleRadius="2rem"
+							circleBackground="var(--button-blue)"
+							textColor='var(--white)'
+							onClick={handleSubmit}
+						/>
```
Comment (vladbyk89) [inline src/view/pages/statement/components/SimilarStatementsSuggestion/StepOneStatementInput.tsx:148]: Could you reduce the number of props you pass down to the button? Some things can be by default.
The button width should be responsive with a maximum width.

File: src/view/pages/statement/components/header/StatementHeader.tsx [hunk 2]
```diff
@@ -82,6 +82,9 @@ const StatementHeader: FC<Props> = ({
 	const titleFontSize = calculateFontSize(title, 16, 25);
 	const isAdmin = statement?.creatorId === user?.uid;
 
+	// Checks if enableNavigationalElements is allowed
+	const enableNavigationalElements = statement?.statementSettings?.enableNavigationalElements;
```
Comment (vladbyk89) [inline src/view/pages/statement/components/header/StatementHeader.tsx:86]: Did you talk to @talyaron about this?

**Review by vladbyk89 — CHANGES_REQUESTED** [review]
Good start for the assignment. Please do not mix the tasks. 
Create separated PR's for each assignment.

File: src/view/pages/statement/components/header/StatementHeader.tsx (hunk 2 above)
Comment (ShakedEdr1) [reply src/view/pages/statement/components/header/StatementHeader.tsx:86]: Sorry for the delayed response, I had some problems with the internet, i'll send him a message, thanks for reviewing

Comment (vladbyk89) [reply src/view/pages/statement/components/header/StatementHeader.tsx:86]: He won't be available until Sunday

Comment (ShakedEdr1) [reply src/view/pages/statement/components/header/StatementHeader.tsx:86]: Alright so I'll talk to him when he returns

File: src/controllers/db/statements/getSimilarstatements.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ export const findSimilarStatements = async (
 		const isProd = false;
 
 		if (isProd) {
-			return prodEndPoint;
+			return localEndPoint;
```
Comment (vladbyk89) [inline src/controllers/db/statements/getSimilarstatements.tsx:16]: Revert these changes and merge the origin/dev to this branch. It was solved there.

File: src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx [hunk 4]
```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
...
 									onClick={() => setShowModal(false)}
 								>
-									<ArrowLeftIcon />
+									<CloseIcon />
 								</button>
-								<img src={illustration02} alt='Similarities illustration' />
+								<img src={illustration01}
+									alt='Similarities illustration'
+									style={{ width: '8rem' }} />
 							</div>
```
Comment (vladbyk89) [inline src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx:129]: Consider refactoring this into a separate function that handles rendering based on the current step. The current implementation is difficult to read and maintain.

File: src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx [hunk 5]
```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
...
+						) : (currentStep === 2 ? (
+							<div className='similarities__header__top__stepThree'>
+								<img
+									src={illustration02}
+									alt='Another illustration'
+									style={{ width: '19rem' }}
+								/>
+								<button
+									className='similarities__header__top__stepThree__arrowButtonStepThree'
+									onClick={() => setShowModal(false)}
```
Comment (vladbyk89) [inline src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx:153]: Clicking the back arrow currently closes the modal, but it should navigate to the previous step instead. Could you adjust the behavior so that it moves back a step rather than closing the modal?

File: src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx [hunk 6]
```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
...
-						<div className='similarities__header__types'>
-							<div className='type'>
-								<AddQuestionIcon />
-								<h2 className={isQuestion ? 'marked' : 'unmarked'}>Question</h2>
+						) : (currentStep === 2 ? (
+							<div className='similarities__header__top__stepThree'>
+								<img
+									src={illustration02}
+									alt='Another illustration'
+									style={{ width: '19rem' }}
```
Comment (vladbyk89) [inline src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx:149]: Could you use vw or % for the image size instead of rem? This will help ensure the image scales better with the viewport size.

File: src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx [hunk 7]
```diff
@@ -114,36 +115,53 @@ export default function SimilarStatementsSuggestion({
...
+									<ArrowLeftIcon />
+								</button>
 							</div>
+						) : null))
+					}
+						<div className={`similarities__header__types  ${currentStep === 2 ? 'hidden' : ''}`}>
+							{/* <div className='type'>
+							    <AddQuestionIcon />
+								<h2 className={isQuestion ? 'marked' : 'unmarked'}>Question</h2>
+							</div> */}
```
Comment (vladbyk89) [inline src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx:164]: Please remove the commented lines from the code

**Review by vladbyk89 — CHANGES_REQUESTED** [review]

File: src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx (hunk 4 above)
Comment (ShakedEdr1) [reply src/view/pages/statement/components/SimilarStatementsSuggestion/SimilarStatementsSuggestion.tsx]: Do the recent changes i made answer your request?

File: src/view/pages/statement/components/header/StatementHeader.tsx (hunk 2 above)
Comment (talyaron) [reply src/view/pages/statement/components/header/StatementHeader.tsx:86]: I am here...

**Review by talyaron — APPROVED** [review]
Thanks. looks good
